contrib/babelfishpg_tsql/src/tsqlIface.cpp

```diff
@@ -3741,6 +3743,20 @@ makeInsertBulkStatement(TSqlParser::Dml_statementContext *ctx)
 		{
 			stmt->db_name = pstrdup(downcase_truncate_identifier(db_name.c_str(), db_name.length(), true));
 		}
+
+		/* create a list of columns to insert into */
+		if (!column_list.empty())
+		{
+			for (size_t i = 0; i < column_list.size() - 1; i++)
+			{
+				if (column_list[i]->simple_column_name())
+					column_refs << ::stripQuoteFromId(column_list[i]->simple_column_name()->id()) << ", ";
+			}
+			if (column_list[column_list.size() - 1]->simple_column_name())
+				column_refs << ::stripQuoteFromId(column_list[column_list.size() - 1]->simple_column_name()->id());
```

Comment:
Will this also handles column names with special characters?

Reply:
Yes it will.